app/models/customer.rb

```diff
@@ -75,6 +75,36 @@ def require_driver_form?
     !driver_form_content.blank?
   end
 
+  # returns Human readable string of the validity length of driver training
+  def helper_driver_training_validity_length?
```

Comment:
Great additions to the model. One note: these method names shouldn't have a question mark at the end because they're returning a value. The question mark typically denotes that there's a Boolean value that will be returned.